sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java

```diff
@@ -98,5 +98,9 @@ public EventDirectorySteps(
     When(
         "I click on New Task from event tab",
         () -> webDriverHelpers.clickOnWebElementBySelector(NEW_TASK_BUTTON));
+
+    When(
+        "I open the first event",
```

Comment:
add more specific data, because first event can be opened from this directory, but from entities as well if they are linked